plugins/mysql/mysql_disk_by_prefix

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+# -*- python -*-
+
+"""
+=head1 INTRODUCTION
+
+Plugin to monitor MySQL database disk usage per prefix
```

Comment:
I think, it is good idea to add one sentence describing the `prefix` (e.g. *database name prefix*) and maybe mention your example of the convention of prefixing database names with the name of their owner.

Reply:
Done.